tests/rptest/tests/topic_recovery_test.py

```diff
@@ -1154,7 +1154,7 @@ def verify():
 
         wait_until(verify,
                    timeout_sec=timeout.total_seconds(),
-                   backoff_sec=1,
+                   backoff_sec=5,
```

Comment:
> We have upload deadline for segments set to 10 sec by default.
Before  we were waiting just for 6 seconds
Because backoff was 1 sec, 

if the 10 sec default changes will the test start breaking again? i don't want to waste too much time on this if a more robust fix would take a long time to create, but i still believe that tests should not be created in a way that their stability depends on tuning backoff_sec.

Reply:
this seems like a case where a comment in the code would be useful. presumably if it fails again folks should be able to quickly figure out that they need to adjust the backoff seconds. that is definitely not intuitive based on how we design tests in general